src/scheduler/gc_work.rs

```diff
@@ -219,11 +219,16 @@ impl<E: ProcessEdgesWork> GCWork<E::VM> for StopMutators<E> {
 impl<E: ProcessEdgesWork> CoordinatorWork<E::VM> for StopMutators<E> {}
 
 #[derive(Default)]
-pub struct EndOfGC;
+pub struct EndOfGC(pub std::time::Duration);
```

Comment:
I recommend giving the field a name.  It is not very obvious what that duration means when looking at this work packet alone.

Reply:
> You can't give field names inside of a tuple struct in Rust. Have to make it a proper struct.

Yes. That's what I mean.  Make it a struct with named fields.